Source/JavaScriptCore/b3/B3AbstractHeapRepository.h

```diff
@@ -280,6 +280,8 @@ class AbstractHeapRepository {
     FOR_EACH_ABSTRACT_HEAP(ABSTRACT_HEAP_DECLARATION)
 #undef ABSTRACT_HEAP_DECLARATION
 
+    AbstractHeap& WebAssemblyMemory;
```

Comment:
Maybe we should have a macro that declares the aliased heaps? There are some other aliases below.